spec/rails_translation_manager/locale_checker_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe RailsTranslationManager::LocaleChecker do
+  context 'when the locales are valid' do
+    it 'outputs a confirmation' do
+      expect { described_class.new('spec/locales/in_sync/**/*.yml').validate_locales }
+        .to output("Locale files are in sync, nice job!\n").to_stdout
+    end
+
+    it 'returns true' do
+      expect(described_class.new('spec/locales/in_sync/**/*.yml').validate_locales)
+        .to eq(true)
+    end
+  end
+
+  context 'when the locales are not valid' do
+    it 'outputs the report' do
+      expect { described_class.new('spec/locales/*.yml').validate_locales }
+        .to output(
+          <<~OUTPUT.chomp
+
+            \e[31m[ERROR]\e[0m Missing English locales, either remove these keys from the foreign locales or add them to the English locales
```

Comment:
Hmmm, I read your commit message about this being an integration test, but feel a little uncomfortable that changes to the MissingLocales logic will involve changing assertions across multiple test files.

Would this still be a valuable integration test by swapping `.to output(` to `.to contains("[ERROR]")`? That way we still assert that there is an error report, without worrying about the specific test failure strings.

Reply:
Hmm that is a good point, being less granular in our text expectations would definitely be better to guard against future test / logic changes! I've re-jigged the spec and only assert the amount of `[ERRORS]` which crop up which gives us what we want.